torch/_export/converter.py

```diff
@@ -56,7 +56,8 @@ def get_op_overload(node: torch._C.Node):
     ns, op_name = str(schema.name.name).split("::")
     override = schema.name.overload_name
 
-    op_overload_packet = getattr(torch.ops.aten, op_name)
+    op_overload_mod = getattr(torch.ops, ns)
+    op_overload_packet = getattr(op_overload_mod, op_name)
```

Comment:
Maybe we can put a try/except around this block -- if there are any errors with getting the operator then we can have a nicer error message.